CHANGELOG.md

```diff
@@ -15,7 +15,9 @@ All notable changes to this project will be documented in this file.
 
 ### Changed
 
-- Make it easy to test custom Nifi images ([#616])
+- Bump `stackable-operator` from `0.64.0` to `0.70.0` ([#641]).
```

Comment:
The previous stackable-operator versions seems to have been 0.69.3, but there was no earlier changelog  stating a change to that version either so maybe it does not matter and can stay like this

Reply:
Yeah, this is a though one, because our changelogs are maintained manually and often misses entries. I put `0.64.0` in here because that's the version used in the `24.3` release.

Reply:
Yes, makes sense :) maybe I'll update my operator PRs tomorrow as well to reflect that

Reply:
I would prefer just having the _to_ version, especially since there can be other bumps between releases and we can just update the final one.

I have commented here https://github.com/stackabletech/secret-operator/pull/467#discussion_r1674186126